### PR TITLE
git subrepo pull (merge) external/os-autoinst-common

### PR DESCRIPTION
**Pull request by b10n1k**
subrepo:
  subdir:   "external/os-autoinst-common"
  merged:   "eb3e05124"
upstream:
  origin:   "git@github.com:os-autoinst/os-autoinst-common.git"
  branch:   "master"
  commit:   "eb3e05124"
git-subrepo:
  version:  "0.4.6"
  origin:   "???"
  commit:   "???"

### CODE REVIEW

File: external/os-autoinst-common/cpanfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ on 'develop' => sub {
     requires 'Code::TidyAll';
     requires 'Perl::Critic';
     requires 'Perl::Critic::Community';
-    requires 'Perl::Tidy', '== 20230912';
+    requires 'Perl::Tidy', '== 20240202';
```
Comment (okurz) [inline external/os-autoinst-common/cpanfile:20]: Before we pull in that I would like us to sort us the conflict that was created that os-autoinst-common relies on the most recent releases on CPAN whereas in other products we rely on the version in Tumbleweed/Leap

**Review by okurz — CHANGES_REQUESTED** [review]

File: external/os-autoinst-common/cpanfile (hunk 1 above)
Comment (perlpunk) [reply external/os-autoinst-common/cpanfile:20]: Yeah. I created https://github.com/os-autoinst/os-autoinst-common/pull/45 and that passes now, so we're back to version 20230912 after that is merged.

**Comment by okurz** [comment]
@b10n1k https://github.com/os-autoinst/os-autoinst-common/pull/45 merged. You can now pull the subrepo again

**Comment by b10n1k** [comment]
> @b10n1k [os-autoinst/os-autoinst-common#45](https://github.com/os-autoinst/os-autoinst-common/pull/45) merged. You can now pull the subrepo again

Done

**Review by perlpunk — APPROVED** [review]

**Review by okurz — APPROVED** [review]

**Pull request merged**